wcfsetup/install/files/js/3rdParty/redactor2/plugins/WoltLabDropdown.js

```diff
@@ -78,24 +31,12 @@ $.Redactor.prototype.WoltLabDropdown = function() {
 		},
 		
 		_show: function() {
+			return;
```

Comment:
Is this an intentional change?

Reply:
Yes, this was during development and a later commit removed this file entirely.

Reply:
I believe I already was reviewing before that commit appeared :smile: